xapian-letor/tests/Makefile.am

```diff
@@ -48,7 +49,7 @@ BUILT_SOURCES =
 
 if MAINTAINER_MODE
 BUILT_SOURCES += api_all.h api_collated.h $(collated_apitest_sources:.cc=.h) \
- api_generated.cc
+ api_generated.cc api_letor.cc
```

Comment:
It's useful mark the comments you think you've dealt with, but commenting with the emoji sends a notification email each time, so we get a bit bombarded.

Better to use "Add your reaction" (hold the mouse pointer over a comment and some icons appear at the top right - you want the "+" and happy face one.